pyuvdata/uvdata/uvh5.py

```diff
@@ -1018,6 +1081,11 @@ def read_uvh5(
                 strict_uvw_antpos_check,
             )
 
+        # For now, always use current shapes when data is read in, even if the file
```

Comment:
This comment jogged my memory about the UVH5 versioning discussed in #983. Let's decide what scheme we'd like to implement and add it as part of this PR (since this is a significant change!).

Reply:
sounds good!

Reply:
Ok @plaplant, I think this PR is ready for you to take over to implement the uvh5 versioning.